### PR TITLE
feat(Workspaces): inject database credentials into workspace notebooks

### PR DESCRIPTION
**Pull request by cheikhgwane**
Inject workspace database credentials (name, host, port...) into workspace notebooks

### CODE REVIEW

File: hexa/workspaces/tests/test_views.py [hunk 1]
```diff
@@ -90,6 +91,21 @@ def test_workspace_credentials_200(self):
             )
         )
 
+        db_credentials = get_db_server_credentials()
+        db_credentials["host"]
```
Comment (pvanliefland) [inline hexa/workspaces/tests/test_views.py:95]: This has no effect, like the next line

File: hexa/workspaces/views.py [hunk 2]
```diff
@@ -35,6 +36,18 @@ def credentials(request: HttpRequest, workspace_slug: str) -> HttpResponse:
 
     # TODO: Database / Filesystem credentials
 
+    # Database credentials
+    db_credentials = get_db_server_credentials()
+    env.update(
```
Comment (qgerome) [inline hexa/workspaces/views.py:41]: I'd only add the `WORKSPACE_DB_URL` at first. @pvanliefland  WDYT ?

**Review by pvanliefland — CHANGES_REQUESTED** [review]
Almost ok

File: hexa/workspaces/views.py (hunk 2 above)
Comment (pvanliefland) [reply hexa/workspaces/views.py:41]: The individual variables are needed for R - the typical postgres lib that our team uses cannot use a connection string url and need the individual parameters

**Pull request merged**